src/ucs/debug/debug.c

```diff
@@ -564,6 +580,42 @@ static void ucs_debug_show_innermost_source_file(FILE *stream)
 
 #endif /* HAVE_DETAILED_BACKTRACE */
 
+/*
+ * Filter specific functions from the head of the backtrace.
+ */
+void ucs_debug_print_backtrace(FILE *stream, int strip)
+{
+    struct backtrace bckt;
+    struct backtrace_line bckt_line;
+    int i;
+
+    ucs_debug_backtrace_create(&bckt, strip);
+
+    fprintf(stream, "==== backtrace (tid:%7d) ====\n", ucs_get_tid());
+    for (i = 0; backtrace_next(&bckt, &bckt_line); ++i) {
+         fprintf(stream, BACKTRACE_FORMAT(i, &bckt_line));
+    }
+    fprintf(stream, "=================================\n");
+
+    ucs_debug_backtrace_destroy(&bckt);
+}
+
+void ucs_debug_log_backtrace(ucs_log_level_t level, int strip)
```

Comment:
pls put this function in ucs/debug/log.{h,c} like this:
`ucs_log_print_backtrace(level)`
strip argument would be 1 always

Reply:
ok, moved into log.h/.c